Tasks/AzureRmWebAppDeployment/AzureRMUtil.ts

```diff
@@ -221,4 +221,53 @@ async function getAzureRMWebAppID(SPN, webAppName: string, resourceType: string)
     });
 
     return deferred.promise;
+}
+
+/**
+ *  REST request for azure webapp config details. Config details contains virtual application mappings.
+ *  
+ *  @param SPN                 Subscription details
+ *  @param webAppName          Web application name
+ *  @param deployToSlotFlag    Should deploy to slot
+ *  @param slotName            Slot for deployment
+ */
+async function getWebAppConfigDetails(SPN, webAppName: string, resourceGroupName: string, deployToSlotFlag: boolean, slotName: string) {
+
+    var deferred = Q.defer<any>();
+    var accessToken = await getAuthorizationToken(SPN);
+    var slotUrl = deployToSlotFlag ? "/slots/" + slotName : "";
+    var configUrl = armUrl + 'subscriptions/' + SPN.subscriptionId + '/resourceGroups/' + resourceGroupName + '/providers/Microsoft.Web/sites/' + webAppName + slotUrl +  '/config/web?api-version=2015-08-01';
+    tl.debug(tl.loc("Requestingconfigdetails", configUrl));
```

Comment:
If the API Version is same for all the APIs should we have a constant globally and use every where